nautobot/extras/templates/extras/inc/job_tiles.html

```diff
@@ -0,0 +1,72 @@
+{% load helpers %}
+
+{% block extra_styles %}
+    <style>
+        .panel.panel-default {
+            border: none;
+            box-shadow: none;
+        }
+    </style>
+{% endblock %}
+
+<div class="tiles">
+    {% for row in table.page.object_list|default:table.rows %}
+        <article class="tile{% if not perms.extras.run_job or not row.record.runnable %} disabled{% endif %}">
+            <header class="tile-header">
+                <h3>{{ row.record.grouping|bettertitle }}</h3>
+
+                <a href="{% url 'extras:job_run' pk=row.record.pk %}" class="btn btn-primary btn-xs" title="Run/Schedule" {% if not perms.extras.run_job or not row.record.runnable %}disabled="disabled"{% endif %}>
+                    <i class="mdi mdi-play" aria-hidden="true"></i>
+                </a>
+
+                {% comment "TODO(norbert-mieczkowski-codilime): use the same buttons (actions) in tables and tiles when tables support dropdown menus" %}
+                    {% for item in row.items %}
+                        {% if item.0.name == "actions" %}
+                            {{ item.1 }}
+                        {% endif %}
+                    {% endfor %}
+                {% endcomment %}
+                <div class="btn-group">
+                    <button type="button" class="btn btn-default btn-xs dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" title="Show more">
+                        <i class="mdi mdi-dots-horizontal"></i>
+                    </button>
+                    <ul class="dropdown-menu dropdown-menu-right context-menu">
+                        <li><a href="{% url 'extras:job' pk=row.record.pk %}"><i class="mdi mdi-information-outline text-muted"></i>Details</a></li>
+                        <li><a href="{% url 'extras:job_changelog' pk=row.record.pk %}"><i class="mdi mdi-history text-muted"></i>Change log</a></li>
+                        {% if perms.extras.change_job %}
+                            <li><a href="{% url 'extras:job_edit' pk=row.record.pk %}"><i class="mdi mdi-pencil text-muted"></i>Edit</a></li>
```

Comment:
Should this use `text-warning` to match the coloring of "edit" buttons elsewhere?

Reply:
I'm not sure, I personally like the way it looks now better. I'll defer the decision to you and @lampwins though, here's comparison of the two:

| Default | Warning |
| - | - |
| ![edit_default](https://github.com/nautobot/nautobot/assets/117445994/74f3f46e-4361-400e-a1e5-9cc6bb9058f2) | ![edit_warning](https://github.com/nautobot/nautobot/assets/117445994/958aec32-6ca8-4883-96b3-ff81e7f5291d) |

Reply:
Yeah I like the original. This is a detail we can iterate on later when we work to expand this to all tables.